CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Official devRant Issue Tracker Rules
+
+Please follow these rules when you are:
+- Submitting a bug report
+- Asking for new features
+- Asking Questions
+
+
+## Submitting a bug report
+- Use the corresponding issue template
+- Please provide clear steps to reproduce the issue, if the steps are known
+- Make sure to include what platform(s) and version of the app you are able to reproduce the bug on
+
+
+## Asking for new features
+- Use the corresponding issue template
+- Please add "[Feature Request]" or "[Feature]" to the title
+- Please provide a detailed description of the feature, how it works, alternatives, maybe even mock-ups?
+- Before submitting, check if there are any workarounds to this feature
```

Comment:
I'm not sure if this is needed. Just because there is a workaround, I don't think we should discourage people from submitting the feature idea. After it's submitted we can discuss and see if there's a workaround that is good enough to the point where a feature wouldn't make sense.

Reply:
Alright I agree